targets/arm32/cm3/atmel/sam3x/sam3x8e/flashscripts/arduino_due.cmake

```diff
@@ -9,17 +9,17 @@ set(DEVICE_LOCATION
 )
 
 if(bossac)
-    if(WIN32)
+    if(MSYS OR MINGW)
```

Comment:
dangit, well we need to find another cross platform way. I'll merge this for now to get things working. But any help finding a better cross platform way for windows users would be nice